### PR TITLE
update generalize samples

### PR DESCRIPTION
**Pull request by sowu880**
Update generalize samples in read me
All samples passed the tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ To generalize string data type using expression to define the value set mapping
 
 ```json
 {
-  "path": "patient.communication.language",
+  "path": "Patient.communication.language.coding",
```
Comment (moria97) [inline README.md:409]: Seems to be Patient.communication.language.coding.code?

File: README.md [hunk 2]
```diff
@@ -432,11 +432,11 @@ To generalize dateTime, time, date and instant type using expression
...
 ```json
 {
-  "path": "person.birthDate",
+  "path": "Patient.birthDate",
   "method": "generalize",
   "cases":{
-    "$this >= @1990-1-1 and $this <= @2000-1-1": "@1990", 
-     "$this >= @2010-1-1 and $this <= @2020-1-1":"@2010-1-1T00:00:00"
+    "$this >= @1990-01-01 and $this <= @2000-01-01": "@1990", 
+     "$this >= @2010-01-01 and $this <= @2020-01-01":"@2010-01-01"
```
Comment (moria97) [inline README.md:439]: Why we remove the timezone info here?

**Review by moria97 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sowu880) [reply README.md:439]: Type of birthdate is "date" that shall not has time zone.

File: README.md (hunk 1 above)
Comment (sowu880) [reply README.md]: Fixed it, thanks.

File: README.md (hunk 2 above)
Comment (moria97) [reply README.md:439]: Make sense. Thanks

**Review by moria97 — APPROVED** [review]

**Review by ranvijaykumar — APPROVED** [review]

**Pull request merged**